From 1c222372d9cae171236add8e3e08d9198cb39a2d Mon Sep 17 00:00:00 2001
From: Martin Pitt <martin.pitt@ubuntu.com>
Date: Mon, 14 Nov 2011 13:32:32 +0000
Subject: Fix gtk_tree_view_get_tooltip_context() transfer annotation

The default (out) transfer mode is "full", but the passed "model" pointer gets
set to gtk_tree_view_get_model() which is "transfer none". This caused Python
programs to free the model after calling this, causing crashes.
---
diff --git a/gtk/gtktreeview.c b/gtk/gtktreeview.c
index 0160818..c1fe401 100644
--- a/gtk/gtktreeview.c
+++ b/gtk/gtktreeview.c
@@ -16286,7 +16286,8 @@ gtk_tree_view_set_tooltip_cell (GtkTreeView       *tree_view,
  * @x: (inout): the x coordinate (relative to widget coordinates)
  * @y: (inout): the y coordinate (relative to widget coordinates)
  * @keyboard_tip: whether this is a keyboard tooltip or not
- * @model: (out) (allow-none): a pointer to receive a #GtkTreeModel or %NULL
+ * @model: (out) (allow-none) (transfer none): a pointer to receive a
+ *         #GtkTreeModel or %NULL
  * @path: (out) (allow-none): a pointer to receive a #GtkTreePath or %NULL
  * @iter: (out) (allow-none): a pointer to receive a #GtkTreeIter or %NULL
  *
--
cgit v0.9.0.2
